front-end-master/landing_page/index.html

```diff
@@ -0,0 +1,217 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<!-- Required meta tags -->
+	<meta charset="utf-8">
```

Comment:
We probably should add more <meta/> tags to enhance SEO. Don't worry about it now. We will create new work action items to take care of it later.

https://www.wordstream.com/meta-tags
https://moz.com/blog/seo-meta-tags